nox/sessions.py

```diff
@@ -208,12 +208,64 @@ def _run(self, *args, env=None, **kwargs):
             kwargs.setdefault("external", "error")
 
         # If we aren't using a virtualenv allow all external programs.
-        if not isinstance(self.virtualenv, VirtualEnv):
+        if not isinstance(self.virtualenv, (CondaEnv, VirtualEnv)):
+            kwargs["external"] = True
+
+        # Allow the global conda command from conda environments.
+        if isinstance(self.virtualenv, CondaEnv) and args[0] == "conda":
```

Comment:
Hrm special casing seems weird here. Maybe we can make `*Env` classes have a `whitelisted_globals` property?

Reply:
Good idea. Done.